.github/workflows/docker-scheduled.yml

```diff
@@ -7,23 +7,18 @@ on:
   workflow_dispatch: # Allow manual trigger
 
 jobs:
-  generate-matrix:
-    runs-on: ubuntu-latest
-    outputs:
-      matrix: ${{ steps.set-matrix.outputs.matrix }}
-    steps:
-      - uses: actions/checkout@v5
-      - name: Generate matrix
-        id: set-matrix
-        run: ruby script/gh/docker-tags.rb --matrix
-
-  build:
-    needs: generate-matrix
+  build-dev:
+    uses: ./.github/workflows/docker.yml@dev
+    with:
+      branch: dev
+      tag: dev
+    secrets: inherit
+  build-release-candidate:
     strategy:
       matrix: ${{ fromJson(needs.generate-matrix.outputs.matrix) }}
       fail-fast: false
-    uses: ./.github/workflows/docker.yml
+    uses: ./.github/workflows/docker.yml@release/16.6
     with:
-      branch: ${{ matrix.branch }}
-      tag: ${{ matrix.tag }}
-    secrets: inherit
\ No newline at end of file
+      branch: release/16.6
```

Comment:
The comment "This reference is updated through devkit" would fit well somewhere here.